sbol_utilities/helper_functions.py

```diff
@@ -357,3 +357,9 @@ def collector(obj: sbol3.Identified):
     with cached_references(doc):
         doc.traverse(collector)
     return outgoing
+
+def find_feature(sbol3: sbol3_file,f: feature,seq: sequence) -> sbol3.Feature:
```

Comment:
Can you please add a docstring that documents the behavior of this function, following the same pattern as the other functions in the file?

It also appears that your type hints are incorrect; does this run correctly when you test it?